### ISSUE TITLE
Actual size is smaller than estimated size, it is not calculated correctly

### ISSUE THREAD

**Issue by kishikawakatsumi** [kishikawakatsumi/IBPCollectionViewCompositionalLayout#114]
Actual size is smaller than estimated size, it is not calculated correctly.

|iOS 13|iOS 12|
|:-:|:-:|
|![Simulator Screen Shot - iPhone Xs - 2019-09-17 at 02 21 47](https://user-images.githubusercontent.com/40610/64979086-e9a2f680-d8f1-11e9-843b-3978265ea222.png)|![Simulator Screen Shot - iPhone Xs - 2019-09-17 at 02 21 58](https://user-images.githubusercontent.com/40610/64979093-ef98d780-d8f1-11e9-8838-065c8057ac57.png)|

**Comment by codyrobb** [comment]
Is this screenshot showing an orthogonal scrolling section up top with too much trailing padding? I am having a similar issue, but I am using absolute sizes rather than estimated sizes. 

I am going to dig into it a bit. I'd love to be able to use this in my project.

**Comment by kishikawakatsumi** [comment]
> Is this screenshot showing an orthogonal scrolling section up top with too much trailing padding?

Yes, this orthogonal section layout has an estimated size, but the actual size is smaller than estimated. But total width doesn't shrink correctly yet. 

Can you share your code to reproduce your problem? I'm not sure this and yours are the same or not. @codyrobb
